netutils/config/compliance.py

```diff
@@ -9,6 +9,7 @@
     "cisco_ios": parser.IOSConfigParser,
     "cisco_nxos": parser.NXOSConfigParser,
     "cisco_aireos": parser.AIREOSConfigParser,
+    "ciscos350": parser.IOSConfigParser,
```

Comment:
Can we subclass `parser.S300ConfigParser` from  `parser.IOSConfigParser` and change `ciscos350` to `cisco_s300` to match netmiko's defintion of this.

Reply:
Sounds good, the napalm module is s350... ugh... updating now